### ISSUE TITLE
Voice Feature for chat 

### ISSUE THREAD

**Issue by PVxDeepak** [Dheerajjha451/CancerDetection#5]
Not an issue just Add voice features for chat  for quick, as it is cancer detection platform so for some emergency case it would be nice, i think if some  user is  using websites as we know there are in such case every minute is important . however you have already have planned to add this features.

**Comment by Dheerajjha451** [comment]
It sounds good. Do you have any example or want to implement this feature?

**Comment by PVxDeepak** [comment]
i am currently going to learn web speech api and after that gone create some basic you can say toy applications basics for understanding its syntax and structures so i sawed your project on discord as i am scrolling there so i think i can add this in your web app. i think it can be easily implemented as you probably using some llm model for generating message so we just need to pass the user voice command texts to your input field and thats it after the your model magic will over there which you already made.first i need to understand code base and state mangement for that components of chatbot field

**Comment by Dheerajjha451** [comment]
@PVxDeepak  Yeah, understand the codebase first, and if you need any help, ping me. Apart from that, for text generation, I am using Meta LLM. Whenever you want to add the features, provide the feature description and video or screenshot of the feature in a PR.

**Comment by PVxDeepak** [comment]
its good to use my practical skills here after learning.i have forked your repo. gone to understand first codebase,i would do pr request after completing it, gone test in  my  local machine  and some in mobile devices.
